### ISSUE TITLE
usage examples - hide filters also hides get results

### ISSUE THREAD

**Issue by balmas** [alpheios-project/components#558]
if you have hidden the filters on the usage examples, then the get results button is hidden too. that should always been shown

**Comment by irina060981** [comment]
Then I suggest to do it this way:
1. word was selected  - but now usage examples were retrieved
![image](https://user-images.githubusercontent.com/12377640/57962183-a2aeef80-7957-11e9-9a14-dce7b123a628.png)
![image](https://user-images.githubusercontent.com/12377640/57962192-ae9ab180-7957-11e9-8e37-04a564c13781.png)

2. Get results without filters
![image](https://user-images.githubusercontent.com/12377640/57962202-ca9e5300-7957-11e9-8630-41cd2423700a.png)

![image](https://user-images.githubusercontent.com/12377640/57962204-d4c05180-7957-11e9-8acc-c97fb78e2c38.png)

3. Filter by the author/textwork
![image](https://user-images.githubusercontent.com/12377640/57962228-f3bee380-7957-11e9-978f-0f09f326c722.png)

![image](https://user-images.githubusercontent.com/12377640/57962239-fae5f180-7957-11e9-91a7-0727039c8971.png)

**Comment by irina060981** [comment]
merged and closed

**Comment by monzug** [comment]
there was a discussion about this show/hide filters in #356 between Irina and me

**Comment by balmas** [comment]
reading for testing in texts-beta
embed-lib 1.0.0.16
webext 3.0.0.37

**Comment by monzug** [comment]
I still have some troubles with this navigation. The hide/show filters does take effect only after clicking on Get results. if I click on Show filter before getting results, I do end up on the same page as by clicking hide filters, except for the Filter by author/work that is grayed out so not selectable. I would suggest to remove the show/hide filters at this level as it is not necessary. Maybe it can be displayed at the same time as Filter these results by Author and/or Work.
an other couple of thing: 1) hide filters includes also the sorting. is it ok to use filters also for sorting?
2) with the current design, there is an intermediate panel generated just after clicking Get results. I could not get a screenshot because it's too fast (I can see the Get results button grayed out) and then the final panel is generated.

**Comment by irina060981** [comment]
> The hide/show filters does take effect only after clicking on Get results. if I click on Show filter before getting results, I do end up on the same page as by clicking hide filters, except for the Filter by author/work that is grayed out so not selectable. I would suggest to remove the show/hide filters at this level as it is not necessary. Maybe it can be displayed at the same time as Filter these results by Author and/or Work.

Will fix it

> hide filters includes also the sorting. is it ok to use filters also for sorting?

I have made it this way - to reduce space

> with the current design, there is an intermediate panel generated just after clicking Get results. I could not get a screenshot because it's too fast (I can see the Get results button grayed out) and then the final panel is generated

yes, I know what you talk about - it is the state when results are generating (that's why the button Get result is disabled), but filters are not ready - so show filters works a little early - I will fix it

**Comment by irina060981** [comment]
Fixed 1 and 3
@balmas and @monzug , what do you think about sorting - do we need additional switch?

Merged

**Comment by monzug** [comment]
I just pointed out to check if it's ok to include sorting into filtering.

**Comment by balmas** [comment]
>what do you think about sorting - do we need additional switch?
I think it's okay for now.

**Comment by irina060981** [comment]
then I will close it

**Comment by balmas** [comment]
can test in texts-beta embed-lib 1.0.0.17

**Comment by monzug** [comment]
Now hide and show filters make sense. Thanks!
tested in Chrome and Firefox  and mobile, build 1.0.0.17
Just realize that if I do double click and go to examples and get results for the same word two times in a row, the second time the author drop-down is empty.  on iphone xr, the dropdown for Authors is not populated the second time I go to examples for the same word
see screenshot
![second-x-examples](https://user-images.githubusercontent.com/41396793/58439843-77897600-80a4-11e9-8088-702877397f1d.JPG)

**Comment by balmas** [comment]
>@balmas, I don't think it's critical for release but it should get fixed soon

Agree. This is reported as new issue #591.  

Can we close this one @monzug ?

**Comment by monzug** [comment]
Absolutely. Will do when I get back home
